### PR TITLE
verminvolk - the small version of WILDKIN

### PR DESCRIPTION
**Pull request by Slorbin**
## About The Pull Request

adds 'verminvolk', a race of kobold-size anthromorphs (WILDKIN) for those who want to play something fluffy and small instead of scaly and small.

The sprites are slightly remixed kobold sprites, with some better arm and leg separation that kobolds don't have so you can add markings.

I had to remake all the markings - offsetting the markings wouldn't have worked, because of how the proportions are. Most of the non-lizard markings made it over, since this is more meant for rodent or mustelid species or such.

They're a shunned race, like kobolds, meaning they have the same jobs available to them.

Their stats are:

male: -2 str, 1 per, 0 int, -1 con, 1 end, 2 speed
female: -3 str, 1 per, 2 int, -1 con, 0 end,  2 speed

I opted to make them an entirely different race instead of making it a drop-down selectable for anthros because:

1. it's way easier this way
2. there's enough changes to them that it warrants its own species like elf to dark elf

## Why It's Good For The Game

lets you play small FURRIES. like SHORT SHORT furries.

not just 'short' as in those earlier controversial sprites, like PINT SIZED. They're kobold-sized and a bit stocky like them too, due to literally being slightly modified kobold sprites.

I got tired of waiting for the guy we're commissioning for 'short' sprites, and his sprites are only going to be a little shorter not kob sized. we can update it with better sprites later if we get them. but for now? kobold works.

here's a few images of them standing next to the other races (goblin, kobold, dwarf, human, elf) and such

linedup
vermvolk1

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/furry/anthromorphsmall.dm [hunk 1]
```diff
@@ -0,0 +1,148 @@
+/mob/living/carbon/human/species/anthromorph/small
+	race = /datum/species/anthromorph/small
+
+/datum/species/anthromorph/small
```
[the remark this answers is not in the document]
Comment (DeerJesus) [reply code/modules/mob/living/carbon/human/species_types/furry/anthromorphsmall.dm:4]: Okay, I’ll change the inheritance.

[the remark this answers is not in the document]
Comment (DeerJesus) [reply code/modules/mob/living/carbon/human/species_types/furry/anthromorphsmall.dm:4]: Should I also make the markings have less inheretence? Currently they’re bodymarkings/small/(marking)

**Comment by Hmeister-real** [comment]
I cant wait to dropkick little runts down cliffs when this gets merged

**Comment by TheRealRatMan** [comment]
Love it! Need this merged rn

**Pull request merged**